### PR TITLE
CSYNC-3238

### PR DESCRIPTION
**Pull request by stefaniaropotica**
- add desktop-sync client in alfresco realm for Dektop Sync apps to use

### CODE REVIEW

**Comment by stefaniaropotica** [comment]
Hi @rgauss and @jamalkm 

Please review my change. It's for https://issues.alfresco.com/jira/browse/CSYNC-3238.

Thank you!

File: helm/alfresco-identity-service/alfresco-realm.json [hunk 1]
```diff
@@ -980,6 +980,59 @@
...
+        "saml.client.signature": "false",
+        "tls.client.certificate.bound.access.tokens": "false",
+        "saml.authnstatement": "false",
+        "display.on.consent.screen": "false",
+        "saml.onetimeuse.condition": "false"
+      },
+      "authenticationFlowBindingOverrides": {},
+      "fullScopeAllowed": true,
+      "nodeReRegistrationTimeout": -1,
+      "defaultClientScopes": [
```
Comment (jamalkm) [inline helm/alfresco-identity-service/alfresco-realm.json]: You don't need to add `defaultClientScopes` and `optionalClientScopes` as these are the defaults.

Comment (stefaniaropotica) [reply helm/alfresco-identity-service/alfresco-realm.json]: Hi @jamalkm,
I added a new commit with the requested changes. Thank you!

**Comment by rgauss** [comment]
I have done a brief test of:
1. Manually imported the client JSON into a running AIMS 1.1 instance
2. Obtained an access token via password grant using the `desktop-sync` client id
3. Used that token to authenticate to a backend service that's configured with the `alfresco` client id (APS' `runtime-app-definitions`)
and things seemed to work fine.

We've created https://issues.alfresco.com/jira/browse/AUTH-434 for the Platform Services team to give this a thorough review and understand any implications of introducing this in AIMS 1.3.

File: helm/alfresco-identity-service/alfresco-realm.json [hunk 2]
```diff
@@ -980,6 +980,47 @@
       "useTemplateScope": false,
       "useTemplateMappers": false
     },
+    {
+      "id": "da2b61de-bd1d-47d1-8106-2fb93cd6b0f2",
+      "clientId": "desktop-sync",
```
Comment (rgauss) [inline helm/alfresco-identity-service/alfresco-realm.json]: We'll want to standardize on an `alfresco-` prefix for our clients to distinguish from clients that customers might add to the same realm.

Comment (stefaniaropotica) [reply helm/alfresco-identity-service/alfresco-realm.json]: Added the prefix, thank you!

**Pull request closed without merging**